### DISCUSSION TITLE
Sumatra pdf freezes every time I update my tex file in wined10

### DISCUSSION THREAD

**Discussion by codezat** [sumatrapdfreader/sumatrapdf#3275] [General]
Hello,
I am running version 3.4.6 64 bit and v10.3 of winedt (64-bit). Every time I update my text file and hit pdftexify, things freeze for  about 5-10s seconds - sumatra pdf clears its window and then reloads.  Everything works after that until I recompile the file the next time. This is quite frustrating. Is there a way around the problem?
Thank you in advance!

**Comment by GitHubRulesOK** [comment]
generally thats how pdf previewers work they release the pdf for however long editor compilation (pdftex/latex/xetex/pdftexify) takes then auto reloads the same freshly built page. 
For small files it can seem instantaneous but there is a threshold around 32MB so then tipping over will seem like a step change.
It would seem either the compilation cycle is slow (it can in some cases be 3 or 4 passes to compile bibtex) or there is a complexity to the generated file that requires (as often the case with PDF) a 100% file analysis before pre-viewing.

If file is around or over 32MB you can try Acrobat / Foxit but no guarantee they would be faster.
